base/libgit2/blob.jl

```diff
@@ -4,15 +4,24 @@ function Base.length(blob::GitBlob)
     return ccall((:git_blob_rawsize, :libgit2), Int64, (Ptr{Void},), blob.ptr)
 end
 
+"""
+    rawcontent(blob::GitBlob) -> Array
+
+Fetch the *raw* contents of the [`GitBlob`](@ref) `blob`. This is a read-only
+`Array` containing the contents of the blob, which may be binary or may be ASCII
+`String` data.
```

Comment:
I'm confused a bit by this wording. Is it always just raw bytes which may or may not be valid ASCII? If it's only sometimes ASCII, is that worth noting? Under what circumstances do each of these outcomes occur?